### PR TITLE
Create german.json

### PR DESCRIPTION
**Pull request by markusmarkusz**
Added some parts of the german translation

### CODE REVIEW

File: locale/core/german.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	"SECONDS": "Sekunden",
+	"SECOND": "Sekunde",
+	"SECOND_SHORT": "s",
+	"MINUTES": "Minuten",
+	"MINUTE": "Minute",
+	"MINUTE_SHORT": "m",
+	"HOURS": "Stunden",
+	"HOUR": "Stunde",
+	"HOUR_SHORT": "h",
```
Comment (Nayruden) [inline locale/core/german.json]: Should this be 'S'? This is a one-character way to refer to hours, so in English, "12h" would mean 12 hours. I realize that 's' is taken by 'seconds', so does 'S' make more sense here? In context, in German, "12S 15m 19s" would mean 12 hours, 15 minutes, 19 seconds.

File: locale/core/german.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	"DAYS": "Tage",
+	"DAY": "Tag",
+	"DAY_SHORT": "d",
```
Comment (Nayruden) [inline locale/core/german.json]: Should be 'T'?

File: locale/core/german.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	"WEEKS": "Wochen",
+	"WEEK": "Woche",
+	"WEEK_SHORT": "w",
+	"MONTHS": "Monate",
+	"MONTH": "Monat",
+	"MONTH_SHORT": "M",
+	"YEARS": "Jahre",
+	"YEAR": "Jahr",
+	"YEAR_SHORT": "y",
```
Comment (Nayruden) [inline locale/core/german.json]: Should be 'J'?

File: locale/core/german.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+	"NUMBER": "Nummer",
+
+	"COMMAND_ARG_MESSAGE": "\"{COMMANDNAME}\" Fehler bei Argument #{ARGNUM}: {MESSAGE}",
+	"COMMAND_UNKNOWN": "Unbekannter Befehl '{COMMANDNAME}'",
+
+	"ARG_NOT_SPECIFIED": "Es wurden nicht genügend Argumente angegeben.",
+	"ARG_CANNOT_PARSE": "Cannot parse '{GIVEN}' to {TYPE}",
+	"ARG_INCORRECT_TYPE": "Not a {TYPE} (given '{GIVEN}')",
```
Comment (Nayruden) [inline locale/core/german.json]: No translations for ARG_CANNOT_PARSE and ARG_INCORRECT_TYPE?

**Comment by Nayruden** [comment]
Very helpful, thank you! Though, we are at a _very_ early stage of ULX4 development, so we will need your help in the future as more phrases get added. Can we contact you in the future regarding German translations?

I made a few comments on your translations; please take a look at them before I merge it.

**Comment by markusmarkusz** [comment]
I hadn't add the short forms (w, m, M, s, ...) because I thought that
they weren't so important. I'll add them later.
Can you give me an example message for ARG_CANNOT_PARSE and
ARG_INCORRECT_TYPE? Then I can translate it better.

And yes you can contact me for future translations to German.

File: locale/core/german.json (hunk 4 above)
Comment (markusmarkusz) [inline locale/core/german.json]: Give me an example sentence. Then I can better translate it ^^.

Comment (Nayruden) [inline locale/core/german.json]: Cannot parse: "Cannot parse 'bob' to number" or "Cannot parse 'table: 0x7fe400512690' to string"

Incorrect type: "Not a number (given 'bob')" or "Not a string (given 'table: 0x7fe400512690')"

Does that make sense?

**Pull request merged**